onnx/common/status.cc

```diff
@@ -40,6 +40,8 @@ std::string Status::ToString() const {
     result += "[CheckerError]";
   } else if (StatusCategory::OPTIMIZER == state_->category) {
     result += "[OptimizerError]";
+  } else if (StatusCategory::INFERENCE == state_->category) {
+	result += "[InferenceError]";
```

Comment:
I would suggest "ShapeInferenceError" in a user-facing message, since Inference is an overloaded term

Reply:
But type inference also covered in the function :). So, TypeShapeInference? if you want.